repository.yaml

```diff
@@ -2,4 +2,6 @@ templates:
   - name: TypeScript - Starter
     path: ./typescript-starter
   - name: JavaScript - Starter
-    path: ./javascript-starter
\ No newline at end of file
+    path: ./javascript-starter
+  - name: Python - Starter (unstable)
```

Comment:
any alternatives to `unstable` we could use?

Reply:
Experimental or Preview?

Reply:
```suggestion
  - name: Python - Starter (preview)
```